ReactSkia/core_modules/RSkKeyboardObserver.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+* Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+*
+* Use of this source code is governed by a BSD-style license that can be
+* found in the LICENSE file.
+*/
+#include "sdk/NotificationCenter.h"
+#include "RSkKeyboardObserver.h"
+#include "ReactSkia/utils/RnsLog.h"
+
+using namespace std;
+
+namespace facebook {
+namespace react {
+
+RSkKeyboardObserver::RSkKeyboardObserver(
+    const std::string &name, 
+    std::shared_ptr<CallInvoker> jsInvoker,
+    Instance *bridgeInstance)
+    : RSkEventEmitter(name, jsInvoker, bridgeInstance){}
+
+void RSkKeyboardObserver::KeyBoardEventHandler(std::string eventName) {
+    RNS_LOG_NOT_IMPL;
+}
+
+void RSkKeyboardObserver::startObserving() {
```

Comment:
we may be having an issue here .
startObserving and stopObserving should probably have eventName as argument .
Ex: If app adds listener for two events keyboardWillShow and keyboardWillHide. Instead of adding two listeners we may end up adding 2*2 times. Right ?

Reply:
It is implemented initially w.r.t RSKTveventHandler. But this is not tested , as we Created separate ticket for it for keyboard APis.
So I will revert these changes. So that it can be analyzed properly & implemented.